doc/user_guide/transform.rst

```diff
@@ -702,9 +702,10 @@ last_value    None       Assigns a value from the last data object in the curren
 nth_value     Number     Assigns a value from the nth data object in the current sliding window frame. If no such object exists, assigns ``null``. Requires a non-negative integer parameter that indicates the offset from the start of the window frame. This operation must have a corresponding entry in the `fields` parameter array.
 ============  =========  =========================================================================================================================================================================================================================================================================================================================
 
-Notice that when we use an aggregation function within an aggregate transform, the data objects are collapsed.
-However, when we use an aggregation function within a window transform, the data objects are augmented.
-It's by augmenting, rather than collapsing, that we can compute statistics, such as `z-scores`_.
+Notice that using an aggregate transform is like using an SQL ``GROUP BY``:
+the input data objects are replaced by the output data objects, which contain the results of applying the aggregation function.
+However, using a window transform is like broadcasting the results to the input data objects.
+By using a window transform, and preserving the input data objects, we can compute statistics, such as `z-scores`_.
 For example, consider the following time series of stock prices:
```

Comment:
Does this address your original comment @jakevdp?

Reply:
I don't really understand the connection here between window functions and broadcasting. To me they seem like distinct concepts: for example, you cannot compute a convolution through broadcasting except via a very contrived process, but you can easily express a convolution via a window transform. I like your description elsewhere of window transforms as aggregates computed in the neighborhood of each row of the data.

Regarding group by & aggregation, I think this is also a confusing comparison. An altair aggregate can express the same thing as a groupby followed by an aggregation (if you specify the ``groupby`` parameter in the transform) but in general aggregation is not equivalent to a group-by.

Reply:
I think we're out of step with our comments and commits. I've removed the references to `GROUP BY` and broadcasting, as well as the original references to collapsed and augmented. My most recent commit incorporated your suggested amendments.

Reply:
I've managed to get into a tangle with PRs and MRs on GitHub before. Sorry for the confusion!

Reply:
Ah, I see. Sorry about that!